lib/class-wp-theme-json-gutenberg.php

```diff
@@ -1307,7 +1307,13 @@ private static function remove_insecure_settings( $input ) {
 					esc_attr( esc_html( $preset['name'] ) ) === $preset['name'] &&
 					sanitize_html_class( $preset['slug'] ) === $preset['slug']
 				) {
-					$value           = $preset[ $preset_metadata['value_key'] ];
+					$value = null;
+					if ( isset( $preset_metadata['value_key'] ) ) {
+						$value = $preset[ $preset_metadata['value_key'] ];
+					} elseif ( isset( $preset_metadata['value_func'] ) ) {
```

Comment:
Would we want to check `is_callable` instead of `isset` for functions? Maybe both?